datafusion/CHANGELOG.md

```diff
@@ -19,6 +19,94 @@
 
 # Changelog
 
+## [12.0.0](https://github.com/apache/arrow-datafusion/tree/12.0.0) (2022-09-09)
```

Comment:
I do not understand why there are no breaking changes listed here. I spent time adding the "api change" label ([link](https://github.com/apache/arrow-datafusion/pulls?q=is%3Apr+is%3Aclosed+label%3A%22api+change%22))

Reply:
I merged lastest from master to pick up the latest commits and generated the changelog again, and this time there are breaking changes. Strange.